src/contour/display/TerminalDisplay.cpp

```diff
@@ -566,9 +577,9 @@ void TerminalDisplay::onBeforeSynchronize()
 
 void TerminalDisplay::createRenderer()
 {
-    // Require(!_renderTarget);
+    Require(!_renderTarget);
+    // Require(!_renderer);
```

Comment:
What is the correct check for this condition, before we were checking that renderer exist

Reply:
this commenting-out was a left-over of my failed attempt to fix sizing on fractional-scaling - until I realized that Qt's devicePixelRatio() returns different values, based on time (during startup)